yt_dlp/extractor/youtube.py

```diff
@@ -649,6 +649,8 @@ def _call_api(self, ep, query, video_id, fatal=True, headers=None,
 
         data = {'context': context} if context else {'context': self._extract_context(default_client=default_client)}
         data.update(query)
+        if self.po_token:
```

Comment:
should be in _extract_player_response imo

Reply:
I think this would work?
```diff
diff --git a/yt_dlp/extractor/youtube.py b/yt_dlp/extractor/youtube.py
index 224c9b988..6a4f7aadf 100644
--- a/yt_dlp/extractor/youtube.py
+++ b/yt_dlp/extractor/youtube.py
@@ -3734,6 +3734,9 @@ def _extract_player_response(self, client, video_id, master_ytcfg, player_ytcfg,
         yt_query = {
             'videoId': video_id,
         }
+
+        if self.po_token:
+            yt_query['serviceIntegrityDimensions'] = {'poToken': self.po_token}
 
         default_pp = traverse_obj(
             INNERTUBE_CLIENTS, (_split_innertube_client(client)[0], 'PLAYER_PARAMS', {str}))
```